File: app/src/main/java/com/neome/api/home/drawer/sig/SigAddressBook.java
// neome.ai API - do not change
//

package com.neome.api.home.drawer.sig;

import com.neome.api.nucleus.base.Types.AnyPrefixKey;
import com.neome.api.home.base.dto.DtoUserAddrBookContact;
import com.neome.api.home.base.dto.DtoUserAddrBookOtherContact;

import java.util.Map;

import com.neome.api.nucleus.base.sig.SigVersion;

@SuppressWarnings("unused")
public class SigAddressBook extends SigVersion
{
  public Map<AnyPrefixKey, DtoUserAddrBookContact[]> candidateMap;

  public DtoUserAddrBookOtherContact[] othersList;
}

File: app/src/main/java/com/neome/api/home/drawer/sig/SigAddressBook.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.home.drawer.sig

import com.neome.api.nucleus.base.Types.AnyPrefixKey
import com.neome.api.home.base.dto.DtoUserAddrBookContact
import com.neome.api.home.base.dto.DtoUserAddrBookOtherContact
import com.neome.api.nucleus.base.sig.SigVersion

interface SigAddressBook : SigVersion
{
  val candidateMap: Map<AnyPrefixKey, Array<DtoUserAddrBookContact>>
  val othersList: Array<DtoUserAddrBookOtherContact>
}